common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -747,6 +737,21 @@
                     self.isMouseFocus = false;
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
+
+                    self.subtitlesEl.attr('aria-label', gettext('Video transcript: activating an item in this group will spool the video to the corresponding time point.')) // jshint ignore:line
+                        .find('.transcript-title')
+                            .html(gettext('Video transcript'));
+
+                    self.subtitlesEl.find('.transcript-start')
```

Comment:
@clrux Since this portion of the code is new, I do think you need to be escaping the text being set here (in case someone translates these strings to have characters that need to be escaped). I believe you want to call the jquery "text" method instead of "html" method. 

http://api.jquery.com/text/